### DISCUSSION TITLE
submit API return message

### DISCUSSION THREAD

**Discussion by elerik** [OP-TED/eForms-SDK#274] [Ideas]
Hi 
what do you think. Is it possible that you will add required version of sdk in return message ?
https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/submit
response body: 
`{
  "code": "notice.SDK_NOT_FOUND",
  "message": "No compatible SDK version found for featureVersion=eforms-sdk-1.1.6",
  "showDetails": false
}`

add something like 
`
...
"sended": "1.1.6",
"required":"1.2.0",
...
`

or just this.
`{
  "code": "notice.SDK_NOT_FOUND",
  "message": "No compatible SDK version found for featureVersion=eforms-sdk-1.1.6. Required version=eforms-sdk-1.4.0",
  "showDetails": false
}`

Or move information about current SDK at top at https://docs.ted.europa.eu/home/eforms/preview/index.html In middle of page its little bit lost :)

**Comment by rouschr** [comment]
Hello,
note that eNotices2 is developed separately, for issues or questions concerning this project use this form:
https://enotices2.ted.europa.eu/help/contact
